Makefile

```diff
@@ -1,6 +1,6 @@
 # Copyright Contributors to the Open Cluster Management project
 
-REGISTRY ?= quay.io/rhibmcollab
+REGISTRY ?= quay.io/open-cluster-management
```

Comment:
Should we use `open-cluster-management` repo? For local development I usually use my personal quay repo. Last I recall, I dont think we have push access w/ our default accounts to `open-cluster-management` repo.

Reply:
I think we aren't technically supposed to be using rhibmcollab and wanted to match it with where our images are officially stored, with the expectation that anyone who wants to build our repo will push to their own registry. @nathanweatherly WDYT?

Reply:
I've reverted the name change